### PR TITLE
PLT-3418, PLT-2950, PLT-3375, PLT-3382, PLT-3446, PLT-3465: Multiple UI Improvements

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey guys have a look, changes include:
- PLT-3418 - Improving icons for system console
- PLT-2950 - Making the get team invite link modal a bit more usable and adding it back again
- PLT-3375 - Adding brackets around some text in about modal
- PLT-3382 - Replacing glyphicons with fontawsome icons (A bunch of files)
- PLT-3446 - Increasing opacity of help text
- PLT-3465 - Making the full width view the default view and improvements to posts and its controls
- There are also some improvements to the help text for message display, auto toggle previews and the preview feature for preview snippet for links.

Can we setup a test server for this, want to show the post and its controls before this goes in.

### CODE REVIEW

File: webapp/components/get_link_modal.jsx [hunk 1]
```diff
@@ -28,6 +29,13 @@ export default class GetLinkModal extends React.Component {
         this.props.onHide();
     }
 
+    copyLinkOnClick() {
+        $(ReactDOM.findDOMNode(this.refs.textarea)).on('click', function copyLinkOnClick() {
```
Comment (jwilander) [inline webapp/components/get_link_modal.jsx]: Few things:
1. If the ref is an HTML element you don't need `ReactDOM.findDOMNode()` you can just access it directly
2. This event listener never gets removed, can you remove it in `componntWillUnmount()`?
3. This event listener looks like a bit of a hack, what's the purpose of it?

File: webapp/components/create_post.jsx [hunk 2]
```diff
@@ -503,7 +503,7 @@ class CreatePost extends React.Component {
         }
 
         let centerClass = '';
-        if (this.state.centerTextbox) {
+        if (!this.state.centerTextbox) {
```
Comment (jwilander) [inline webapp/components/create_post.jsx]: Why's the logic being reversed here? Doesn't seem to make sense to add a `center` class when the state says not to center

File: webapp/components/get_link_modal.jsx (hunk 1 above)
Comment (asaadmahmoodspin) [inline webapp/components/get_link_modal.jsx]: Top copy the link when someone clicks on the input.

**Comment by asaadmahmoodspin** [comment]
Updated.

**Comment by jwilander** [comment]
+1

**Comment by it33** [comment]
These changes are AWESOME!!!

In particular, love the new design that reduces the right margin in center channel. Huge improvement over previous versions. 

Fantastic work!

image

**Comment by it33** [comment]
Proposed tweaks to help text: 

Proposed tweaks to help text: 

1. Message Display, help text: 

"Standard View" shows messages in Mattermost's default formatting, including profile pictures, different sized fonts when users apply formatting, and the names of users shown in a line above message posts. . 

"Compact View" shows messages without profile pictures, without different sized message fonts, and with the names of users shown on the same line as message posts. 

If you are on a slow internet connection, consider using "Compact View" since it does not require the downloading of profile pics to read messages. 

To make views even more compact, consider using the "Auto Collapse Preview" setting to not automatically render previews of images in links.

2. Auto Collapse Preview, help text: 

"On" helps save vertical space in channels by automatically collapsing previews of images generated from web links included in messages. You can click the uncollapse icon if you decide you want to see a collapsed preview. 

"Off" gives you more information about web links posted in messages by allowing previews from weblinks to be displayed by default. You can click the collapse icon on any preview if you decide you no longer want to see it.

**Comment by jwilander** [comment]
+1

**Comment by enahum** [comment]
+1

**Comment by enahum** [comment]
@lfbrock is this PR ready to merge?

**Comment by lfbrock** [comment]
yes, +1 (tested on AM's spinmint)

**Pull request merged**